pkg/api/constants/extensions.go

```diff
@@ -5,6 +5,8 @@ const (
 	ExtCatalogPrefix     = "/_catalog"
 	ExtOciDiscoverPrefix = "/_oci/ext/discover"
 	// zot specific extensions.
-	ExtSearchPrefix  = "/_zot/ext/search"
-	FullSearchPrefix = RoutePrefix + ExtSearchPrefix
+	ExtSearchPrefix     = "/_zot/ext/search"
+	ExtAuthQueryPrefix  = "/_zot/ext/authquery"
```

Comment:
This should be "mgmt"

Reply:
mgmt/config, mgmt/<something-else>

https://github.com/opencontainers/distribution-spec/tree/main/extensions
^ think we can do that